### DISCUSSION TITLE
Best tool for web3 development

### DISCUSSION THREAD

**Discussion by BrutKode** [community/community#39428] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Which is the best tool for **Web3 Development**?

 - [ ] Hardhat
 - [ ] Ethers
 - [ ] Foundry
 - [ ] Brownie

What would you use, if you had to work on a hackathon with a very tight time schedule and also a very configurable project? Which is begginer friendly?

**Comment by david-engelmann** [comment]
I'd recommend a Hardhat + Web3.js combo. You can get setup relatively quickly and both are extremely well documented. Hardhat will make getting on testnets and mainnets simple!
